spec/features/users/users_balances_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'Users::Balances', type: :feature do
+  let(:user) { create(:user) }
+  let(:balance) { Users::Balance.new(user).balance }
+  let!(:user_transactions) { create_list(:transaction, 10, user: user) }
+  let(:expected_balance) { user_transactions.map { |transaction| transaction.amount.to_i }.sum }
```

Comment:
@marderer why didn't you use 
```ruby
user_transactions.sum(:amount).to_i
```
instead of:
```ruby
user_transactions.map { |transaction| transaction.amount.to_i }.sum
```
as was mentioned in https://github.com/marderer/personal-finance/pull/25#discussion_r174750041?

Reply:
@AleksSenkou 
```user_transactions.sum(:amount).to_i```
It isn't working. Amount is a string in my database